### ISSUE TITLE
Support symfony 5 and Sonata 4

### ISSUE THREAD

**Issue by eerison** [sonata-project/SonataPageBundle#1416]
## Support `Symfony 5` and `Sonata 4`

It's a sub task from #1410

**Note:** this issue is blocked by 
- [x] #1096 
- [x] #1415 
- [x] #1420 
- [x] #1433 
- [x] #1475 
- [x] #1417 
- [x] #1477
- [x] #1442 
- [x] #1491
- [x] ~~#1454~~

**Comment by jordisala1991** [comment]
The Symfony Runtime is not mandatory to enable support for Symfony 5.4 and 6.0. IMO it should stay at its own issue (and will be solved too for 4.0)

**Comment by eerison** [comment]
> The Symfony Runtime is not mandatory to enable support for Symfony 5.4 and 6.0. IMO it should stay at its own issue (and will be solved too for 4.0)

Hmmm I updated one project from 4.4 to 5.4 and it force me to use runtime, do you have any application running without symfony runtime?

**Comment by eerison** [comment]
> The Symfony Runtime is not mandatory to enable support for Symfony 5.4 and 6.0. IMO it should stay at its own issue (and will be solved too for 4.0)

Yeah I got what do you mean!

**Comment by VincentLanglet** [comment]
I'll close this one in favor of the only missing check which is https://github.com/sonata-project/SonataPageBundle/issues/1410
